modules/islandora_entities_csv_import/includes/utilities.inc

```diff
@@ -82,6 +98,13 @@ function islandora_entities_build_scholar_object($scholar, $collection_pid) {
   $fedora_object->relationships->add(FEDORA_MODEL_URI, 'hasModel', 'islandora:personCModel');
   $fedora_object->relationships->add(FEDORA_RELS_EXT_URI, 'isMemberOfCollection', $collection_pid);
   $new_fedora_object = islandora_add_object($fedora_object);
+  if ($new_fedora_object && !$collection_object) {
```

Comment:
I'm confused here. Why are we allowing orphan objects to be ingested?

Reply:
I thought this was the expected legacy behaviour (but we now issue a warning). It should not happen when coming from the form, but if someone were to call the function directly, we keep doing what we always did. I am happy to remove this, though...